Nur in flrig-1.3.53-TMD710: .project.
diff -crB flrig-1.3.53/src/include/rigs.h flrig-1.3.53-TMD710/src/include/rigs.h
*** flrig-1.3.53/src/include/rigs.h	2020-11-16 16:09:28.000000000 +0100
--- flrig-1.3.53-TMD710/src/include/rigs.h	2020-12-25 09:57:53.143729407 +0100
***************
*** 80,85 ****
--- 80,86 ----
  #include "KX3.h"
  #include "PCR1000.h"
  #include "RAY152.h"
+ #include "TMD710.h"
  #include "TS140.h"
  #include "TS450S.h"
  #include "TS480HX.h"
***************
*** 167,172 ****
--- 168,174 ----
  extern RIG_KX3			rig_KX3;		// 58
  extern RIG_PCR1000		rig_PCR1000;	// 59
  extern RIG_RAY152		rig_RAY152;		// 60
+ extern RIG_TMD710		rig_TMD710;
  extern RIG_TS140		rig_TS140;		// 61
  extern RIG_TS450S		rig_TS450S;		// 62
  extern RIG_TS480HX		rig_TS480HX;	// 63
diff -crB flrig-1.3.53/src/Makefile.am flrig-1.3.53-TMD710/src/Makefile.am
*** flrig-1.3.53/src/Makefile.am	2020-11-16 16:10:59.000000000 +0100
--- flrig-1.3.53-TMD710/src/Makefile.am	2020-12-25 08:40:48.959865737 +0100
***************
*** 189,194 ****
--- 189,195 ----
  	rigs/KX3.cxx \
  	rigs/PCR1000.cxx \
  	rigs/RAY152.cxx \
+ 	rigs/TMD710.cxx \
  	rigs/TS140.cxx \
  	rigs/TS450S.cxx \
  	rigs/TS480HX.cxx \
***************
*** 355,360 ****
--- 356,362 ----
  	include/support.h \
  	include/threads.h \
  	include/trace.h \
+ 	include/TMD710.h \
  	include/TS140.h \
  	include/TS450S.h \
  	include/TS480HX.h \
diff -crB flrig-1.3.53/src/Makefile.in flrig-1.3.53-TMD710/src/Makefile.in
*** flrig-1.3.53/src/Makefile.in	2020-12-02 13:43:32.000000000 +0100
--- flrig-1.3.53-TMD710/src/Makefile.in	2020-12-25 08:44:21.656722360 +0100
***************
*** 146,152 ****
  	rigs/IC7610.cxx rigs/IC7800.cxx rigs/IC7851.cxx \
  	rigs/IC9100.cxx rigs/IC9700.cxx rigs/IC7700.cxx \
  	rigs/ICF8101.cxx rigs/KENWOOD.cxx rigs/K2.cxx rigs/K3.cxx \
! 	rigs/KX3.cxx rigs/PCR1000.cxx rigs/RAY152.cxx rigs/TS140.cxx \
  	rigs/TS450S.cxx rigs/TS480HX.cxx rigs/TS480SAT.cxx \
  	rigs/TS570.cxx rigs/TS590S.cxx rigs/TS590SG.cxx rigs/TS790.cxx \
  	rigs/TS850.cxx rigs/TS870S.cxx rigs/TS890S.cxx rigs/TS940S.cxx \
--- 146,152 ----
  	rigs/IC7610.cxx rigs/IC7800.cxx rigs/IC7851.cxx \
  	rigs/IC9100.cxx rigs/IC9700.cxx rigs/IC7700.cxx \
  	rigs/ICF8101.cxx rigs/KENWOOD.cxx rigs/K2.cxx rigs/K3.cxx \
! 	rigs/KX3.cxx rigs/PCR1000.cxx rigs/RAY152.cxx rigs/TMD710.cxx rigs/TS140.cxx \
  	rigs/TS450S.cxx rigs/TS480HX.cxx rigs/TS480SAT.cxx \
  	rigs/TS570.cxx rigs/TS590S.cxx rigs/TS590SG.cxx rigs/TS790.cxx \
  	rigs/TS850.cxx rigs/TS870S.cxx rigs/TS890S.cxx rigs/TS940S.cxx \
***************
*** 214,219 ****
--- 214,220 ----
  	rigs/flrig-KENWOOD.$(OBJEXT) rigs/flrig-K2.$(OBJEXT) \
  	rigs/flrig-K3.$(OBJEXT) rigs/flrig-KX3.$(OBJEXT) \
  	rigs/flrig-PCR1000.$(OBJEXT) rigs/flrig-RAY152.$(OBJEXT) \
+ 	rigs/flrig-TMD710.$(OBJEXT) \
  	rigs/flrig-TS140.$(OBJEXT) rigs/flrig-TS450S.$(OBJEXT) \
  	rigs/flrig-TS480HX.$(OBJEXT) rigs/flrig-TS480SAT.$(OBJEXT) \
  	rigs/flrig-TS570.$(OBJEXT) rigs/flrig-TS590S.$(OBJEXT) \
***************
*** 313,318 ****
--- 314,320 ----
  	rigs/$(DEPDIR)/flrig-K2.Po rigs/$(DEPDIR)/flrig-K3.Po \
  	rigs/$(DEPDIR)/flrig-KENWOOD.Po rigs/$(DEPDIR)/flrig-KX3.Po \
  	rigs/$(DEPDIR)/flrig-PCR1000.Po rigs/$(DEPDIR)/flrig-RAY152.Po \
+ 	rigs/$(DEPDIR)/flrig-TMD710.Po \
  	rigs/$(DEPDIR)/flrig-TS140.Po rigs/$(DEPDIR)/flrig-TS2000.Po \
  	rigs/$(DEPDIR)/flrig-TS450S.Po rigs/$(DEPDIR)/flrig-TS480HX.Po \
  	rigs/$(DEPDIR)/flrig-TS480SAT.Po rigs/$(DEPDIR)/flrig-TS570.Po \
***************
*** 622,628 ****
  	rigs/IC7610.cxx rigs/IC7800.cxx rigs/IC7851.cxx \
  	rigs/IC9100.cxx rigs/IC9700.cxx rigs/IC7700.cxx \
  	rigs/ICF8101.cxx rigs/KENWOOD.cxx rigs/K2.cxx rigs/K3.cxx \
! 	rigs/KX3.cxx rigs/PCR1000.cxx rigs/RAY152.cxx rigs/TS140.cxx \
  	rigs/TS450S.cxx rigs/TS480HX.cxx rigs/TS480SAT.cxx \
  	rigs/TS570.cxx rigs/TS590S.cxx rigs/TS590SG.cxx rigs/TS790.cxx \
  	rigs/TS850.cxx rigs/TS870S.cxx rigs/TS890S.cxx rigs/TS940S.cxx \
--- 624,631 ----
  	rigs/IC7610.cxx rigs/IC7800.cxx rigs/IC7851.cxx \
  	rigs/IC9100.cxx rigs/IC9700.cxx rigs/IC7700.cxx \
  	rigs/ICF8101.cxx rigs/KENWOOD.cxx rigs/K2.cxx rigs/K3.cxx \
! 	rigs/KX3.cxx rigs/PCR1000.cxx rigs/RAY152.cxx \
! 	rigs/TMD710.cxx rigs/TS140.cxx \
  	rigs/TS450S.cxx rigs/TS480HX.cxx rigs/TS480SAT.cxx \
  	rigs/TS570.cxx rigs/TS590S.cxx rigs/TS590SG.cxx rigs/TS790.cxx \
  	rigs/TS850.cxx rigs/TS870S.cxx rigs/TS890S.cxx rigs/TS940S.cxx \
***************
*** 702,708 ****
  	include/rig_io.h include/rigpanel.h include/serial.h \
  	include/socket.h include/socket_io.h include/status.h \
  	include/support.h include/threads.h include/trace.h \
! 	include/TS140.h include/TS450S.h include/TS480HX.h \
  	include/TS480SAT.h include/TS570.h include/TS590S.h \
  	include/TS590SG.h include/TS790.h include/TS850.h \
  	include/TS870S.h include/TS890S.h include/TS940S.h \
--- 705,711 ----
  	include/rig_io.h include/rigpanel.h include/serial.h \
  	include/socket.h include/socket_io.h include/status.h \
  	include/support.h include/threads.h include/trace.h \
! 	include/TMD710.h include/TS140.h include/TS450S.h include/TS480HX.h \
  	include/TS480SAT.h include/TS570.h include/TS590S.h \
  	include/TS590SG.h include/TS790.h include/TS850.h \
  	include/TS870S.h include/TS890S.h include/TS940S.h \
***************
*** 1018,1023 ****
--- 1021,1028 ----
  	rigs/$(DEPDIR)/$(am__dirstamp)
  rigs/flrig-RAY152.$(OBJEXT): rigs/$(am__dirstamp) \
  	rigs/$(DEPDIR)/$(am__dirstamp)
+ rigs/flrig-TMD710.$(OBJEXT): rigs/$(am__dirstamp) \
+ 	rigs/$(DEPDIR)/$(am__dirstamp)
  rigs/flrig-TS140.$(OBJEXT): rigs/$(am__dirstamp) \
  	rigs/$(DEPDIR)/$(am__dirstamp)
  rigs/flrig-TS450S.$(OBJEXT): rigs/$(am__dirstamp) \
***************
*** 1233,1238 ****
--- 1238,1244 ----
  @AMDEP_TRUE@@am__include@ @am__quote@rigs/$(DEPDIR)/flrig-KX3.Po@am__quote@ # am--include-marker
  @AMDEP_TRUE@@am__include@ @am__quote@rigs/$(DEPDIR)/flrig-PCR1000.Po@am__quote@ # am--include-marker
  @AMDEP_TRUE@@am__include@ @am__quote@rigs/$(DEPDIR)/flrig-RAY152.Po@am__quote@ # am--include-marker
+ @AMDEP_TRUE@@am__include@ @am__quote@rigs/$(DEPDIR)/flrig-TMD710.Po@am__quote@ # am--include-marker
  @AMDEP_TRUE@@am__include@ @am__quote@rigs/$(DEPDIR)/flrig-TS140.Po@am__quote@ # am--include-marker
  @AMDEP_TRUE@@am__include@ @am__quote@rigs/$(DEPDIR)/flrig-TS2000.Po@am__quote@ # am--include-marker
  @AMDEP_TRUE@@am__include@ @am__quote@rigs/$(DEPDIR)/flrig-TS450S.Po@am__quote@ # am--include-marker
***************
*** 2441,2446 ****
--- 2447,2466 ----
  @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
  @am__fastdepCXX_FALSE@	$(AM_V_CXX@am__nodep@)$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(flrig_CPPFLAGS) $(CPPFLAGS) $(flrig_CXXFLAGS) $(CXXFLAGS) -c -o rigs/flrig-RAY152.obj `if test -f 'rigs/RAY152.cxx'; then $(CYGPATH_W) 'rigs/RAY152.cxx'; else $(CYGPATH_W) '$(srcdir)/rigs/RAY152.cxx'; fi`
  
+ rigs/flrig-TMD710.o: rigs/TMD710.cxx
+ @am__fastdepCXX_TRUE@	$(AM_V_CXX)$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(flrig_CPPFLAGS) $(CPPFLAGS) $(flrig_CXXFLAGS) $(CXXFLAGS) -MT rigs/flrig-TMD710.o -MD -MP -MF rigs/$(DEPDIR)/flrig-TMD710.Tpo -c -o rigs/flrig-TMD710.o `test -f 'rigs/TMD710.cxx' || echo '$(srcdir)/'`rigs/TMD710.cxx
+ @am__fastdepCXX_TRUE@	$(AM_V_at)$(am__mv) rigs/$(DEPDIR)/flrig-TMD710.Tpo rigs/$(DEPDIR)/flrig-TMD710.Po
+ @AMDEP_TRUE@@am__fastdepCXX_FALSE@	$(AM_V_CXX)source='rigs/TMD710.cxx' object='rigs/flrig-TMD710.o' libtool=no @AMDEPBACKSLASH@
+ @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
+ @am__fastdepCXX_FALSE@	$(AM_V_CXX@am__nodep@)$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(flrig_CPPFLAGS) $(CPPFLAGS) $(flrig_CXXFLAGS) $(CXXFLAGS) -c -o rigs/flrig-TMD710.o `test -f 'rigs/TMD710.cxx' || echo '$(srcdir)/'`rigs/TMD710.cxx
+ 
+ rigs/flrig-TMD710.obj: rigs/TMD710.cxx
+ @am__fastdepCXX_TRUE@	$(AM_V_CXX)$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(flrig_CPPFLAGS) $(CPPFLAGS) $(flrig_CXXFLAGS) $(CXXFLAGS) -MT rigs/flrig-TMD710.obj -MD -MP -MF rigs/$(DEPDIR)/flrig-TMD710.Tpo -c -o rigs/flrig-TMD710.obj `if test -f 'rigs/TMD710.cxx'; then $(CYGPATH_W) 'rigs/TMD710.cxx'; else $(CYGPATH_W) '$(srcdir)/rigs/TMD710.cxx'; fi`
+ @am__fastdepCXX_TRUE@	$(AM_V_at)$(am__mv) rigs/$(DEPDIR)/flrig-TMD710.Tpo rigs/$(DEPDIR)/flrig-TMD710.Po
+ @AMDEP_TRUE@@am__fastdepCXX_FALSE@	$(AM_V_CXX)source='rigs/TMD710.cxx' object='rigs/flrig-TMD710.obj' libtool=no @AMDEPBACKSLASH@
+ @AMDEP_TRUE@@am__fastdepCXX_FALSE@	DEPDIR=$(DEPDIR) $(CXXDEPMODE) $(depcomp) @AMDEPBACKSLASH@
+ @am__fastdepCXX_FALSE@	$(AM_V_CXX@am__nodep@)$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(flrig_CPPFLAGS) $(CPPFLAGS) $(flrig_CXXFLAGS) $(CXXFLAGS) -c -o rigs/flrig-TMD710.obj `if test -f 'rigs/TMD710.cxx'; then $(CYGPATH_W) 'rigs/TMD710.cxx'; else $(CYGPATH_W) '$(srcdir)/rigs/TMD710.cxx'; fi`
+ 
  rigs/flrig-TS140.o: rigs/TS140.cxx
  @am__fastdepCXX_TRUE@	$(AM_V_CXX)$(CXX) $(DEFS) $(DEFAULT_INCLUDES) $(INCLUDES) $(flrig_CPPFLAGS) $(CPPFLAGS) $(flrig_CXXFLAGS) $(CXXFLAGS) -MT rigs/flrig-TS140.o -MD -MP -MF rigs/$(DEPDIR)/flrig-TS140.Tpo -c -o rigs/flrig-TS140.o `test -f 'rigs/TS140.cxx' || echo '$(srcdir)/'`rigs/TS140.cxx
  @am__fastdepCXX_TRUE@	$(AM_V_at)$(am__mv) rigs/$(DEPDIR)/flrig-TS140.Tpo rigs/$(DEPDIR)/flrig-TS140.Po
***************
*** 3430,3435 ****
--- 3450,3456 ----
  	-rm -f rigs/$(DEPDIR)/flrig-KX3.Po
  	-rm -f rigs/$(DEPDIR)/flrig-PCR1000.Po
  	-rm -f rigs/$(DEPDIR)/flrig-RAY152.Po
+ 	-rm -f rigs/$(DEPDIR)/flrig-TMD710.Po
  	-rm -f rigs/$(DEPDIR)/flrig-TS140.Po
  	-rm -f rigs/$(DEPDIR)/flrig-TS2000.Po
  	-rm -f rigs/$(DEPDIR)/flrig-TS450S.Po
***************
*** 3608,3613 ****
--- 3629,3635 ----
  	-rm -f rigs/$(DEPDIR)/flrig-KX3.Po
  	-rm -f rigs/$(DEPDIR)/flrig-PCR1000.Po
  	-rm -f rigs/$(DEPDIR)/flrig-RAY152.Po
+ 	-rm -f rigs/$(DEPDIR)/flrig-TMD710.Po
  	-rm -f rigs/$(DEPDIR)/flrig-TS140.Po
  	-rm -f rigs/$(DEPDIR)/flrig-TS2000.Po
  	-rm -f rigs/$(DEPDIR)/flrig-TS450S.Po
diff -crB flrig-1.3.53/src/rigs/rigs.cxx flrig-1.3.53-TMD710/src/rigs/rigs.cxx
*** flrig-1.3.53/src/rigs/rigs.cxx	2020-11-16 15:59:19.000000000 +0100
--- flrig-1.3.53-TMD710/src/rigs/rigs.cxx	2020-12-25 08:16:30.026250826 +0100
***************
*** 89,94 ****
--- 89,95 ----
  RIG_KX3			rig_KX3;
  RIG_PCR1000		rig_PCR1000;
  RIG_RAY152		rig_RAY152;
+ RIG_TMD710		rig_TMD710;
  RIG_TS140		rig_TS140;
  RIG_TS450S		rig_TS450S;
  RIG_TS480HX		rig_TS480HX;
***************
*** 177,182 ****
--- 178,184 ----
  	&rig_KX3,		// 58
  	&rig_PCR1000,	// 59
  	&rig_RAY152,	// 60
+ 	&rig_TMD710,
  	&rig_TS140,		// 61
  	&rig_TS450S,	// 62
  	&rig_TS480HX,	// 63
